### PR TITLE
Create pull request template file

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_531/

**Review by nitayneeman — APPROVED** [review]
Nice!

But I'm not sure I will stick to this XD

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   Before creating the PR:
+  =========================
+  
+    - Please make sure your commits are signed,the PR cannot be merged without it.
+      Read more about it here:
+      https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits
+      
+    - If the PR changes/adds something to the UI, make sure it's aligned to the design system specs:
+      https://zeroheight.com/7sjjzhgo2/p/7181b5-tpa-design-system
+
```
Comment (nitayneeman) [inline .github/pull_request_template.md:14]: It could be nice to create a tutorial explaining our CI status checks (and link it here)

Comment (jonathanadler) [reply .github/pull_request_template.md:14]: 🤔  interesting

I'll merge this PR now, but we can add it later

**Pull request merged**